src/renderers/shaders/ShaderChunk/transmission_fragment.glsl.js

```diff
@@ -22,11 +22,12 @@ export default /* glsl */`
 	vec3 n = inverseTransformDirection( normal, viewMatrix );
 
 	vec4 transmission = getIBLVolumeRefraction(
-		n, v, roughnessFactor, material.diffuseColor, material.specularColor, material.specularF90,
+		n, v, material.roughness, material.diffuseColor, material.specularColor, material.specularF90,
```

Comment:
`roughnessFactor` includes the roughness map; `material.roughness` does not.

Similarly for `thicknessFactor`.

Is this change correct?

Reply:
`material.roughness` includes roughness map too because it includes `roughnessFactor`.

https://github.com/mrdoob/three.js/blob/246315740026aecd68684b9a2b0df67d1fabbac7/src/renderers/shaders/ShaderChunk/lights_physical_fragment.glsl.js#L8-L10

Reply:
@sunag Yes, you are correct...

```
meshphysical.glsl.js

	uniform float roughness;

#include <lights_physical_pars_fragment>

	struct PhysicalMaterial {
		vec3 diffuseColor;
		float roughness;
		...

#include <roughnessmap_fragment>

	float roughnessFactor = roughness;
	roughnessFactor *= texelRoughness.g;


#include <lights_physical_fragment>

	PhysicalMaterial material;
	material.roughness = max( roughnessFactor, 0.0525 );
```